Update hot reload documentation to show how DN validation can be skipped (#9079)
opensearch-project/documentation-website: _security/configuration/tls.md

This setting is `false` by default.
{: .note }

After enabling hot reloading, use the Reload Certificates API to replace the expired certificates. The new certificates need to be stored in the same location as the previous certificates in order to prevent any changes to the `opensearch.yml` file. 

> By default, the API expects the old certificates to be replaced with valid certificates issued with the same `Issuer/Subject DN` and `SAN`. This behaviour can be disabled by adding the following settings in `opensearch.yml`:

Reviewer: ```suggestion
By default, the Reload Certificates API expects the old certificates to be replaced with valid certificates issued with the same `Issuer/Subject DN` and `SAN`. This behavior can be disabled by adding the following settings in `opensearch.yml`:
```

> ```

Reviewer: ```suggestion
```yml
```

plugins.security.ssl.http.enforce_cert_reload_dn_verification: false
plugins.security.ssl.transport.enforce_cert_reload_dn_verification: false
```

> 

Reviewer: ```suggestion
{% include copy.html %}

```

> These settings are `true` by default.

Reviewer: ```suggestion

```
